services/mariadb/builder.js

```diff
@@ -7,9 +7,9 @@ const _ = require('lodash');
 module.exports = {
   name: 'mariadb',
   config: {
-    version: '10.3',
-    supported: ['10.6', '10.5', '10.4', '10.3', '10.2', '10.1'],
```

Comment:
Existing versions should not be removed from the supported array. If you remove this, for people who use these versions, Lando will result to `ERROR ==> mariadb version 10.2 is not supported`.

Reply:
@kaszarobert Tormi put versions 10.3 and below into the `legacy` array, so it'd just flash a warning that users are on old versions.

Reply:
@reynoldsalec:  We would want to keep 10.3 in the supported array and remove from the legacy array, right?